docs/blockchain/block-producers/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Block Producers"
+outline: [0,3]
+---
+
+# Block Producers
+
+Block producers (BPs) play a crucial role Antelope networks. Ultra is an Antelope based network that runs with its own protocol changes and differences. Here's an overview of what Block Producers are expected to do.
+
+### Block Production
+
+Block producers are responsible for creating new blocks in the Antelope blockchain.
```

Comment:
in the next bullet points, sometimes 'BPs' is used and sometimes 'Block producers'. it would be more coherent if we use one of them after the first bullet point.